packages/flutter/lib/src/gestures/long_press.dart

```diff
@@ -19,20 +22,36 @@ class LongPressGestureRecognizer extends PrimaryPointerGestureRecognizer {
   /// Consider assigning the [onLongPress] callback after creating this object.
   LongPressGestureRecognizer({ Object debugOwner }) : super(deadline: kLongPressTimeout, debugOwner: debugOwner);
 
+  //is a flag which is used to determine if the longPress action happened
+  bool _longPressAccepted = false;
+
   /// Called when a long-press is recognized.
   GestureLongPressCallback onLongPress;
+  
+  /// Called when long-press is recognized and the pointer stops to contact the screen
```

Comment:
maybe "Called when the pointer stops contacting the screen after the long-press gesture has been recognized."